misc/roundtrip

```diff
@@ -112,8 +114,10 @@ OLD_IFS="$IFS"
 IFS='
 '
 set -x
-for tags in $(find "$UNITS" -name expected.tags); do
-    for name in $(sed -e 's/^\([^	]*\)	.*/\1/' "$tags"); do
+tagfiles="$(find "$UNITS" -name expected.tags)"
```

Comment:
The first and the last double quote characters are needed.

Does the following code do the same?
```
tagfiles=$(find "$UNITS" -name expected.tags)
```

Reply:
yes, doesn't matter